FWCore/Framework/src/Worker.h

```diff
@@ -118,6 +119,11 @@ namespace edm {
     Worker(Worker const&) = delete;             // Disallow copying and moving
     Worker& operator=(Worker const&) = delete;  // Disallow copying and moving
 
+    void clearModule() {
```

Comment:
When will we have a Worker around when its module is gone?

Reply:
I agree it would be cleaner to remove also the Worker. I suppose that cleanup should extend in all places holding a pointer to a Worker, e.g. ProductResolvers.

Maybe such a cleanup could be done in a follow-up PR?

Reply:
Doing the cleanup here or in a follow-up PR is up to you.

Reply:
I created an issue to remind us https://github.com/cms-sw/cmssw/issues/30826.